tests/test_prices.py

```diff
@@ -211,36 +234,57 @@ def test_init_taxedmoney_model_field():
 
 
 def test_init_taxedmoney_model_field_validation():
-    instance = Model(price=TaxedMoney(Money(25, "USD"), Money(30, "USD")))
-    with pytest.raises(ValueError):
+    instance = Model(price=TaxedMoney(Money("25.566", "USD"), Money(30, "USD")))
```

Comment:
I don't understand this change.

Reply:
You're right, I made wrong by testing different condition. 
This test has no sense now - it checks that currency is equal to the field currency, but now currency can be set arbitrary.